tests/e2e/cases/manifests/operators.yml

```diff
@@ -17,5 +17,10 @@ spec:
     spec:
       containers:
         - name: xline-operator
-          image: datenlord/xline-operator:latest
+          image: xline-kv/xline-operator:latest
```

Comment:
Is it possible that `ghcr.io/xline-kv/xline-operator:latest`? just same with `ghcr.io/xline-kv/xline:latest`

Reply:
No, the image is built locally before running this e2e script. So the name does not matter.